tests/layer_tests/tensorflow_tests/test_tf_BinaryOps.py

```diff
@@ -101,20 +103,22 @@ def create_add_placeholder_const_net(self, x_shape, y_shape, op_type):
                              ['Add', 'AddV2', 'Sub', 'Mul', 'Div', 'RealDiv', 'SquaredDifference', 'Pow',
                               'Maximum', 'Minimum', 'Equal', 'NotEqual', 'Mod', 'Greater', 'GreaterEqual', 'Less',
                               'LessEqual', 'LogicalAnd', 'LogicalOr', 'FloorMod', 'FloorDiv',
-                              'Xdivy', 'BitwiseAnd', 'BitwiseOr', 'BitwiseXor', ])
+                              'Xdivy', 'BitwiseAnd', 'BitwiseOr', 'BitwiseXor', 'LeftShift', 'RightShift'])
```

Comment:
please create separate file for testing these operation test_tf_BitwiseShift.py because not this file is too overloaded and we need input data specifically generated for these operations. It will be more manageable among different platforms (ARM, GPU and OSes) this way.

Reply:
Done, please note that the types tested now are related to the types listed in op `evaluate`.
The list of types in `evaluate` has been reduced to minimum, to avoid increasing binary size (it can be expanded in the future if needed). The TF LeftShift/RighShift are currently enabled by CPU fallback to the reference impl, so the tested types are aligned. More types can be added to TF tests with native CPU impl support (or by switching the tested plugin to TEMPLATE).